tools/wasm-sourcemap.py

```diff
@@ -140,7 +140,7 @@ def encode_uint_var(n):
 def append_source_mapping(wasm, url):
   logger.debug('Append sourceMappingURL section')
   section_name = "sourceMappingURL"
-  section_content = encode_uint_var(len(section_name)) + section_name + encode_uint_var(len(url)) + url
+  section_content = encode_uint_var(len(section_name)) + bytes(section_name, encoding="utf-8") + encode_uint_var(len(url)) + bytes(url, encoding="utf-8")
```

Comment:
can you do `section_name.encode()` instread?

Reply:
yep, just did that. thanks for the hint!